Gemfile.lock

```diff
@@ -123,18 +123,18 @@ GEM
       colored2 (~> 3.1)
       nanaimo (~> 0.4.0)
       rexml (>= 3.3.6, < 4.0)
-    xcpretty (0.4.0)
-      rouge (~> 3.28.0)
+    xcpretty (0.3.0)
+      rouge (~> 2.0.7)
```

Comment:
Were these downgrades on purpose? I think snyk will open PRs for upgrades again.

Reply:
This has been downgraded on purpose waiting for https://github.com/xcpretty/xcpretty/pull/398 to be merged. There was a breaking change introduced on `0.4.0` that wasn't compatible with certain ruby versions